lib/elements/footing.rb

```diff
@@ -0,0 +1,35 @@
+module Elements
+  class Footing
+    attr_reader :concrete
```

Comment:
cool if we call it `material`? :)

Reply:
yeah, and congruent with Rebar and RebarHook cases